test.md

```diff
@@ -20,8 +20,45 @@ This subsort contains Auxiliary functions that only used in our KWASM semantics
     syntax Stmt ::= Auxil
 ```
 
-Assertions
-----------
+Reference Interpreter Commands
+------------------------------
+
+TODO: Move this to a separate `EMBEDDER` module?
+
+The official test suite contains some special auxillary instructions outside of the standard Wasm semantics.
+The reference interpreter is a particular embedder with auxillary instructions, specified in [spec interpreter](https://github.com/WebAssembly/spec/blob/master/interpreter/README.md).
+
+### Function Invocation
+
+We allow to `invoke` a function by its exported name in the test code.
+
+```k
+    syntax Auxil ::= "(" "invoke" String ")"
+ // ----------------------------------------
+    rule <k> ( invoke ENAME:String ) => ( call TFIDX ) ... </k>
+         <exports> ... ENAME |-> TFIDX ... </exports>
+```
+
+### Registering Modules
+
+We will reference modules by name in imports.
+`register` is the instruction that allows us to associate a name with a module.
+
+```k
+    syntax Auxil ::= "(" "register" String               ")"
+                   | "(" "register" String TextFormatIdx ")"
+ // --------------------------------------------------------
+    rule <k> ( register S ) => ( register S (NEXT -Int 1) )... </k> // Register last instantiated module.
+         <nextModuleIdx> NEXT </nextModuleIdx>
+         requires NEXT >Int 0
```

Comment:
`r` in `requires` lines up with `l` in `rule`.

Also, let's get rid of the inline comment at the end of line 51.